lib/membrane/testing/pipeline.ex

```diff
@@ -528,4 +530,19 @@ defmodule Membrane.Testing.Pipeline do
   defp combine_results({custom_actions, custom_state}, {actions, state}) do
     {Enum.concat(custom_actions, actions), Map.put(state, :custom_pipeline_state, custom_state)}
   end
+
+  defp validate_options!(options) do
+    allowed_keys_1 = [:mode, :spec, :test_process, :name, :raise_on_child_pad_removed?]
+    allowed_keys_2 = [:module, :custom_args, :test_process, :name]
```

Comment:
```suggestion
    allowed_keys = [:mode, :spec, :test_process, :name, :raise_on_child_pad_removed?]
    allowed_keys_custom_module = [:module, :custom_args, :test_process, :name]
```
And according to spec, `module: :default` can be passed exlicitly